projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/class-wpcom-admin-bar.php

```diff
@@ -20,25 +20,32 @@ class WPCOM_Admin_Bar extends \WP_Admin_Bar {
 		/**
 		 * Site menu
 		 */
-		'wp-admin/'                            => 'https://wordpress.com/home/%home_url%',
-		'wp-admin/plugins.php'                 => 'https://wordpress.com/plugins/%home_url%',
-		'wp-admin/themes.php'                  => 'https://wordpress.com/themes/%home_url%',
+		'wp-admin/'              => 'https://wordpress.com/home/%home_url%',
+		'wp-admin/plugins.php'   => 'https://wordpress.com/plugins/%home_url%',
+		'wp-admin/themes.php'    => 'https://wordpress.com/themes/%home_url%',
 
 		/**
 		 * +New menu
 		 */
-		'wp-admin/post-new.php'                => '/wp-admin/post-new.php?post_type=post&calypsoify=1',
-		'wp-admin/media-new.php'               => 'https://wordpress.com/media/%home_url%',
-		'wp-admin/post-new.php?post_type=page' => 'https://wordpress.com/page/%home_url%',
-		'wp-admin/user-new.php'                => 'https://wordpress.com/people/new/%home_url%',
-
-		/**
-		 * Jetpack
-		 */
-		'wp-admin/post-new.php?post_type=jetpack-testimonial' => 'https://wordpress.com/types/jetpack-testimonial/%home_url%',
-		'wp-admin/post-new.php?post_type=jetpack-portfolio' => 'https://wordpress.com/types/jetpack-portfolio/%home_url%',
```

Comment:
These items are no longer shown when the experiment is enabled. Why is that?

Reply:
Oh, nice find! I think that's a separate issue. It seems that Projects and Testimonials aren't shown at all on Calypso screens.

I've tested it on production and with the control variation and I was able to reproduce it there too.

Reply:
I don't think it's a separate issue. The main issue mentions:
> We should ensure the behaviour is the same on non-Untangled sites with the interface set to Classic/wp-admin.

This isn't true right now since I can see those links on the classic interface.

Reply:
I think that's interpretable. The main issue is about replacing the Calypso links with their WP-Admin equivalent, not fixing every undiscovered bug/inconsistency we find.

 Looking at the Calypso code, it seems that the list of post types is [hardcoded to posts and pages](https://github.com/Automattic/wp-calypso/blob/bb06a5fda62adc6e5c55d9e8e4e499498a090115/client/layout/masterbar/logged-in.jsx#L365).

Even if we do hardcode the links for Projects and Testimonials, the problem won't be completely fixed because on Atomic users can still have CPTs that should be registered in that menu.

In any case, whether we decide to handle it in the same issue or not (I don't think so), it's not blocking this PR because the code lives in Calypso and we'll have to implement the code in there.

Reply:
Please create an issue so that we don't forget about it.